Gems/Atom/Feature/Common/Code/Source/Decals/DecalTextureArrayFeatureProcessor.cpp

```diff
@@ -295,7 +295,7 @@ namespace AZ
 
         void DecalTextureArrayFeatureProcessor::SetDecalMaterial(const DecalHandle handle, const AZ::Data::AssetId material)
         {
-            AZ_PROFILE_FUNCTION(AZ::Debug::ProfileCategory::Renderer);
+            AZ_PROFILE_FUNCTION(Renderer);
```

Comment:
So these AZ_PROFILE_FUNCTION categories are "Renderer", while the ones above in this file is "AzRender"

Reply:
You'll actually find a ton of these discrepancies from before I'm afraid. I'll fix the ones in these files but have made a task that we need to clean-up/consolidate our marker usage later